gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-edit-page/documentation-edit-page.component.scss

```diff
@@ -60,3 +60,13 @@ $typography: map.get(gio.$mat-theme, typography);
     gap: 8px;
   }
 }
+
+::ng-deep table td {
```

Comment:
You must not do this 
deactivate the encapsulation and modify the `tr` `td`, `p` of the entire application. and that only when you load this component, which creates unpredictable changes 😱